### PR TITLE
Create user from different contexts as admin

### PR DESCRIPTION
**Pull request by diogo-costa-bjss**
- Created a single config to handle the creation of user
- Can be used to:  
    - Create a new user (Base)
    - Create a new user inside a unit (Unit)
    - Create a new user inside a team (Team)
- Updated button from `Add new user` in user find page

### CODE REVIEW

File: src/modules/feature-modules/admin/pages/users/user-new.config.ts [hunk 1]
```diff
@@ -1,200 +1,212 @@
...
-
+function summaryParsing(data: StepPayloadType): WizardSummaryType[] {
   const toReturn: WizardSummaryType[] = [];
 
+  let editStepNumber = 1;
+
   toReturn.push(
-    { label: 'Name', value: data.name, editStepNumber: 2  },
-    { label: 'Email', value: data.email, editStepNumber: 1  },
+    { label: 'Email', value: data.email, editStepNumber: editStepNumber++ },
```
Comment (eduarda-meirinhos-bjss) [inline src/modules/feature-modules/admin/pages/users/user-new.config.ts:184]: "Name" must be the first information at summary

File: src/modules/feature-modules/admin/pages/users/user-new.config.ts [hunk 2]
```diff
@@ -1,200 +1,212 @@
...
-          validEmail: true,
-          maxLength: 100
-        }
-      }]
+      parameters: [
+        {
+          id: 'email',
+          dataType: 'text',
+          label: stepsLabels.l1.label,
+          validations: {
```
[the remark this answers is not in the document]
Comment (diogo-costa-bjss) [reply src/modules/feature-modules/admin/pages/users/user-new.config.ts:64]: In Figma there is no description, it was something that was discussed?
image

[the remark this answers is not in the document]
Comment (eduarda-meirinhos-bjss) [reply src/modules/feature-modules/admin/pages/users/user-new.config.ts:64]: Mariana said we should inform the user about character limit, like "Enter an email with a maximum of 100 characters."

File: src/modules/feature-modules/admin/pages/users/user-new.config.ts (hunk 1 above)
Comment (diogo-costa-bjss) [reply src/modules/feature-modules/admin/pages/users/user-new.config.ts:184]: I changed it to maintain the order that the questions appear for the user, talked to Mariana and is now changed on Figma

File: src/modules/feature-modules/admin/pages/users/user-new.config.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (diogo-costa-bjss) [reply src/modules/feature-modules/admin/pages/users/user-new.config.ts:65]: Added in the latest commit, thanks!

**Pull request merged**